docs/book/src/tasks/experimental-features/experimental-features.md

```diff
@@ -0,0 +1,57 @@
+# Experimental Features
```

Comment:
Might be worth to add a list of experimental features available today linking to the specific page